SPECS-EXTENDED/perl-Module-Install-ReadmeFromPod/perl-Module-Install-ReadmeFromPod.spec

```diff
@@ -0,0 +1,196 @@
+Name:           perl-Module-Install-ReadmeFromPod
+Version:        0.30
+Release:        13%{?dist}
+License:        GPL+ OR Artistic
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://metacpan.org/release/Module-Install-ReadmeFromPod
+Source0:        https://cpan.metacpan.org/authors/id/B/BI/BINGOS/Module-Install-ReadmeFromPod-%{version}.tar.gz#/perl-Module-Install-ReadmeFromPod-%{version}.tar.gz
+# Regenerate README in UTF-8
+Patch0:         Module-Install-ReadmeFromPod-0.26-Regenerate-README-in-UTF-8.patch
+# Remove a bogus test that fails on PDF binary files randomly, CPAN RT#130221
+Patch1:         Module-Install-ReadmeFromPod-0.30-Do-not-test-PDF-file-for-new-lines.patch
+BuildArch:      noarch
+BuildRequires:  coreutils
+BuildRequires:  make
+BuildRequires:  perl(Config)
+BuildRequires:  perl(File::Remove)
+BuildRequires:  perl(FindBin)
+BuildRequires:  perl(Module::CoreList)
+BuildRequires:  perl(Module::Install::AuthorRequires) >= 0.02
+BuildRequires:  perl(Module::Install::AutoLicense)
+BuildRequires:  perl(Module::Install::GithubMeta)
+BuildRequires:  perl(Module::Install::Metadata)
+BuildRequires:  perl(inc::Module::Install)
+BuildRequires:  perl(lib)
+BuildRequires:  perl(strict)
+BuildRequires:  perl-generators
+BuildRequires:  perl-interpreter
+BuildRequires:  sed
+# Build script uses lib/Module/Install/ReadmeFromPod.pm
+# Run-time:
+BuildRequires:  perl(base)
+BuildRequires:  perl(Capture::Tiny) >= 0.05
+BuildRequires:  perl(IO::All)
+# Module::Install::Base version from Module::Install in Makefile.PL
+BuildRequires:  perl(Module::Install::Base) >= 1
+BuildRequires:  perl(Pod::Html)
+BuildRequires:  perl(Pod::Man)
+BuildRequires:  perl(Pod::Markdown) >= 2
+BuildRequires:  perl(Pod::Text) >= 3.13
+BuildRequires:  perl(vars)
+BuildRequires:  perl(warnings)
+# Optional run-time:
+%if %{with perl_Module_Install_ReadmeFromPod_enables_pdf}
+BuildRequires:  perl(App::pod2pdf)
+%endif
+# Tests:
+BuildRequires:  perl(File::Path)
+BuildRequires:  perl(File::Temp)
+BuildRequires:  perl(Test::InDistDir)
+BuildRequires:  perl(Test::More) >= 0.47
+%if %{with perl_Module_Install_ReadmeFromPod_enablas_optional_test}
+# Optional tests:
+BuildRequires:  perl(Test::Pod) >= 1.00
+BuildRequires:  perl(Test::Pod::Coverage) >= 1.00
+%endif
+Requires:       perl(:MODULE_COMPAT_%(eval "`perl -V:version`"; echo $version))
+%if %{with perl_Module_Install_ReadmeFromPod_enables_pdf}
+Suggests:       perl(App::pod2pdf)
+%endif
+Requires:       perl(Capture::Tiny) >= 0.05
+Requires:       perl(IO::All)
+# Module::Install::Base version from Module::Install in Makefile.PL
+Requires:       perl(Module::Install::Base) >= 1
+Requires:       perl(Pod::Html)
+Requires:       perl(Pod::Man)
+Requires:       perl(Pod::Markdown) >= 2
+Requires:       perl(Pod::Text) >= 3.13
+
+# Remove under-specified dependencies
+%global __requires_exclude %{?__requires_exclude:%{__requires_exclude}|}^perl\\(Module::Install::Base\\)$
```

Comment:
Why are we removing these dependencies? They are usually required for the package to function properly. Is this something we inherited from Fedora?

Reply:
Yes this is inherited from Fedora